lib/ts/framework/hapi/framework.ts

```diff
@@ -126,17 +126,19 @@ export class HapiResponse extends BaseResponse {
     ) => {
         let now = Date.now();
 
+        const cookieOptions = {
+            isHttpOnly: httpOnly,
+            isSecure: secure,
+            path: path,
+            domain,
+            ttl: expires - now,
+            isSameSite: (sameSite === "lax" ? "Lax" : sameSite === "none" ? "None" : "Strict") as SameSitePolicy,
```

Comment:
Didn't really work for me. 

<img width="1225" alt="image" src="https://github.com/supertokens/supertokens-node/assets/22813027/ef543f63-1a7d-4f34-b7d9-a40448ec6b1a">

Reply:
Right, because in this case you would have to apply it per branch e.g.: `"Lax" as const`, anyway. See [here](https://www.typescriptlang.org/play?#code/C4TwDgpgBAggygEQGJQLxQEQEMDOATAMwygB9MAjfIgbgChaBjAewDsdgowsAnLAWzRQA3rShiouQgC4oACgRZgEAHQsmAd1kBKKAD4oABigB+TAA8qARmIyMFwgCYMO3JklFSFKhgA0tAL50jKzsnDz8DoIi4hJUMvKKKmqaOvpGptjesVDMbBy2lITErrnsWn6BtAQAriwMwACWrFBmslgyQrHSsIhI-jrR4qVMADYqI0wA5m3K7lp0-vStXLx887TL4XwO80A)